astronomy/time_scales_body.hpp

```diff
@@ -12,6 +12,7 @@
 #include "geometry/named_quantities.hpp"
 #include "glog/logging.h"
 #include "numerics/double_precision.hpp"
+#include "quantities/elementary_functions.hpp"
```

Comment:
Why this include?  Shouldn't you get `next.hpp`?

Reply:
`numerics/next.hpp` only has the one for `double`.